README.md

```diff
@@ -0,0 +1,77 @@
+# cinema-app
+
+**What this application is for and what it can do**
+Primitive demonstration program of service of cinema services and its clients
+
+**Technologies used**
+Program based on the use of:
+- Hibernate 
+- Spring framework
+
+**How to run a project**
+Before lunching the programm you need to install on you comp:
+MySQL Workbench (DB for datas working) and Apache Tomcat (to lunch the programm)
+
+To lunch the project you need to copy program **cinema-app-1.0-SNAPSHOT.war** from this server
+and place it in /webapps Tomcat-programm`s on your comp.
+Lunch **/bin/startup.bat** Tomcat-programm/
+In brouser you can see authentication data to start of the program.
+But at first you need to inject previus data in your DB for first authentication.
+So you need to lunch in brouser: **http://localhost:8080/inject**
+You can see "Done!" in brousers window (or "HTTP Status 500 – Internal Server Error" - dont pay your attantion on it)).
+Next step - authentication: lunch in brouser **http://localhost:8080/login** 
+and input "alice@i.ua" as login, "12345678" as password (this person have role "ADMIN")
```

Comment:
It should look like numbered list and much shorter, you should remove target folder, so you don't need to mention it here